rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -22,8 +22,9 @@
 
 from __future__ import print_function
 import os
-
 import rpm
+import six
+import re
```

Comment:
`re` is part of standard library, so it should be grouped with `import os`, `six` is third party library, so it should stay grouped with `import rpm`.